backend/Terraform/modules/cloudFront/variable.tf

```diff
@@ -1,7 +1,7 @@
 variable "s3_bucket_website_index_document" {
   type        = string
   description = "the default file for the static website"
-  default     = "index.html" #change to the real index.html
+  default     = "${path.module}/../../../frontend/posts.html"
```

Comment:
This shouldn't be the path of the repository but the path of the files being uploaded to the bucket (which is only the frontend directory contents